src/Donors/DonorStatisticsQuery.php

```diff
@@ -108,6 +110,91 @@ public function getDonationsCount(): int
         return $query->count('donation.ID');
     }
 
+
+    /**
+     * @unreleased
+     */
+    public function getFirstDonation()
```

Comment:
we should add some unit tests for these statistics

Reply:
Here are some initial tests:
b03ae03e2822991d61bd00348da4c47c0b9eeb34
7f4d9f0ab29e33f150b8cf8f19fc9f11433cf36b